definitions/horizontal_scaling.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Horizontal Scaling
+status: Feedback Appreciated
+category: Concept
+---
+
+## Horizontal Scaling
+
+### What it is
+
+In [cloud computing](https://github.com/cncf/glossary/blob/main/definitions/cloud_computing.md) perspective Horizontal Scaling is type of [scalability](https://github.com/cncf/glossary/blob/main/definitions/scalability.md) whose approach is to add additional instances instead of moving into a larger instance size.
+
+For example, let us consider we have a system of 4GB RAM and we want to increase it's capacity to 16GB RAM, Horizontal Scaling means doing this by making a system of 4 X 4GB RAM rather than switching to a 16GB RAM system.
+
+It is an approach to enhance the performance of the server [node](https://github.com/cncf/glossary/blob/main/definitions/nodes.md) by adding new instances of the server to already existing servers in order to distribute the workload equally. In simple words it aims to decrease the server's load rather than expanding capacity of the individual server.
```

Comment:
I believe "server node" isn't the right description here. Nodes run on a server and you may add more nodes on the same server (@JasonMorgan, correct me if I'm wrong) . Here's my suggested change:

This approach enhances system performance by adding additional instances, or nodes, to an existing server and distributing the workload equally among them. In simple words, horizontal scaling decreases the instance's load rather than expanding its capacity.

As you can see, I introduced "instance" but also refer back to nodes so the reader understands it's the same thing.

Reply:
I'm going to make an edit in line to clean this up. I'm glad we got the definition  of node in here but the usage is a little confusing.